aws/api-gateway/enable-access-logging/metadata.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Can you generate also index file like we did here 5383611fa2f9eaec994bca3eb9b8297e03c1ce1a 
maybe on the way we can auto generate the AVD ID

Reply:
yes thats no problem. This was a demo PR  - the auto gen of ID will be easy when we know the format